permissions/plugin-ecs-publisher.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "ecs-publisher"
+paths:
+- "de/eacg/ecs-publisher"
+developers:
+- "prizrack13"
+- "jthDEV"
```

Comment:
No such user exists in Artifactory. Please review the PR checklist and follow its instructions.

Reply:
plz check again
I have created jthDEV and login https://repo.jenkins-ci.org/webapp/#/home

Reply:
plz check again
I have created jthDEV and login https://repo.jenkins-ci.org/webapp/#/home

Reply:
Username should be lowercase, as that's how it appears in Artifactory, and this is case-sensitive.